src/output.js

```diff
@@ -48,9 +49,26 @@ class Debugger {
 
   title (severity, title, subtitle) {
     if (this.enabled) {
+      const date = new Date();
+      const dateString = chalk.grey(date.toLocaleTimeString());
       const titleFormatted = colors.formatTitle(severity, title);
       const subTitleFormatted = colors.formatText(severity, subtitle);
-      this.log(titleFormatted, subTitleFormatted);
+      const message = `${titleFormatted} ${subTitleFormatted}`
+
+      // In test environment we don't include timestamp
+      if(process.env.NODE_ENV === 'test') {
+        this.log(message);
+        this.log();
+        return;
+      }
+
+      // Make timestamp appear at the end of the line
+      let logSpace = process.stdout.columns - stringLength(message) - stringLength(dateString)
+      if (logSpace <= 0) {
+        logSpace = 10
+      }
```

Comment:
This makes sure there's always a little bit of space between the title message and the date.

<img width="240" alt="screen shot 2017-02-16 at 01 00 59" src="https://cloud.githubusercontent.com/assets/6324199/23001024/70d982d2-f3e3-11e6-9e37-f36c84f8b56f.png">